de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/impexp/BankStatementImportProcess.java

```diff
@@ -0,0 +1,456 @@
+package de.metas.banking.impexp;
+
+import static org.adempiere.model.InterfaceWrapperHelper.newInstance;
+import static org.adempiere.model.InterfaceWrapperHelper.save;
+
+import java.math.BigDecimal;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Properties;
+
+import org.adempiere.ad.dao.IQueryBL;
+import org.adempiere.ad.trx.api.ITrx;
+import org.adempiere.impexp.AbstractImportProcess;
+import org.adempiere.impexp.IImportInterceptor;
+import org.adempiere.util.lang.IMutable;
+import org.compiere.model.I_I_BankStatement;
+import org.compiere.model.ModelValidationEngine;
+import org.compiere.model.X_I_BankStatement;
+
+import de.metas.banking.model.I_C_BankStatement;
+import de.metas.banking.model.I_C_BankStatementLine;
+import de.metas.banking.service.IBankStatementDAO;
+import de.metas.util.Check;
+import de.metas.util.Services;
+import de.metas.util.lang.CoalesceUtil;
+import lombok.NonNull;
+
+/*
+ * #%L
+ * de.metas.banking.base
+ * %%
+ * Copyright (C) 2019 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+public class BankStatementImportProcess extends AbstractImportProcess<I_I_BankStatement>
+{
+
+	private int p_C_BP_BankAccount_ID = 0;
+
+	private void getBankStatementImportProcessParameter()
+	{
+
+		p_C_BP_BankAccount_ID = getParameters().getParameterAsInt("C_BP_BankAccount_ID", -1);
+	}
+
+	@Override
+	public Class<I_I_BankStatement> getImportModelClass()
+	{
+		return I_I_BankStatement.class;
+	}
+
+	@Override
+	public String getImportTableName()
+	{
+		return I_I_BankStatement.Table_Name;
+	}
+
+	@Override
+	protected String getTargetTableName()
+	{
+		return I_C_BankStatement.Table_Name;
+	}
+
+	@Override
+	protected void updateAndValidateImportRecords()
+	{
+
+		final String whereClause = getWhereClause();
+
+		getBankStatementImportProcessParameter();
```

Comment:
vi think u can make `p_C_BP_BankAccount_ID` a local var 
... and u can drop `getBankStatementImportProcessParameter` method and inline it here

Reply:
Done